department-of-veterans-affairs/caseflow
# Save Correspondence Tasks & Upload to eFolder

Resolves [Save Correspondence Tasks & Upload to eFolder](https://jira.devops.va.gov/browse/APPEALS-36279)

# Description
Persists mail task data and "tasks not related to an appeal" data on correspondence intake form submit.

## Acceptance Criteria

## Testing Plan
1. Fill out intake correspondence form, being sure to add data for "mail tasks" and "tasks not related to an appeal".
2. Verify the submit is successful and the above-mentioned data is persisted correctly.

# Frontend
## User Facing Changes
n/a

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
